assets/js/googlesitekit/datastore/site/registry-key.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * core/site data store: registryKey tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+} from 'tests/js/utils';
+import { STORE_NAME } from './constants';
+
+describe( 'core/site registryKey', () => {
+	let registry;
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	describe( 'actions', () => {
+		describe( 'setRegistryKey', () => {
+			it( 'requires the registryKey param', () => {
+				expect( () => {
+					registry.dispatch( STORE_NAME ).setRegistryKey();
+				} ).toThrow( 'registryKey is required.' );
+			} );
+		} );
+	} );
+	describe( 'selectors', () => {
```

Comment:
Breaking these up with a newline is usually what we do, I think it increases readability:

```suggestion

	describe( 'selectors', () => {
```